### PR TITLE
WIP: Recent Shows 

### PR DESCRIPTION
**Pull request by skylarth**
Lakeside shows; 72; Alice in Wonderland

### CODE REVIEW

File: _shows/19_20/freshers_fringe.md [hunk 1]
```diff
@@ -0,0 +1,34 @@
+---
+title: 'Freshers\' Fringe'
```
Comment (wjdp) [inline _shows/19_20/freshers_fringe.md]: Seems upset with this line, try `title: "Freshers' Fringe"`

You can test if the YAML section is valid by pasting into http://www.yamllint.com/

**Comment by BenWoodford76** [comment]
any reason we haven't got a netlify build on this branch?

**Comment by BenWoodford76** [comment]
@samozzy are you happy once netlify has done it's thing?

**Comment by samozzy** [comment]
@BenWoodford76 Oui

**Pull request merged**